Sources/SwiftKotlinFramework/KotlinTokenizer.swift

```diff
@@ -682,7 +978,9 @@ public class KotlinTokenizer: SwiftTokenizer {
                 expression.newToken(.keyword, "invoke")
             ], at: startIndex)
         }
-        return tokens
+        tokens = IdentifiersTransformPlugin.TransformKotlinFunctionCallExpression(tokens)
+
```

Comment:
Same than before, this method should not be dependent on a specific plugin. Move the code in here